README.md

```diff
@@ -0,0 +1,19 @@
+# Cloud Native AI Pipeline
+
+_(Disclaimer: This project is currently under active development, and as such, all source code may not be included in any release. This means that the code is subject to change without notice, and that any information contained within the code should be considered as work in progress.)_
+
+Cloud-native technologies can be applied to artificial intelligence (AI) for scalable
+application in dynamic environment such as public, private and hybrid cloud. But
+it requires cloud native design to break monolithic inference pipeline in several
```

Comment:
It can be more clear to change to "to **decomposes** monolithic inference pipeline **into**...". What do you think?